ui/app/templates/components/auth-info.hbs

```diff
@@ -2,38 +2,38 @@
 <div class="popup-menu-content">
   <div class="box">
     <div class="menu-label">
-      {{auth.authData.displayName}}
+      {{this.auth.authData.displayName}}
     </div>
     <nav class="menu">
       <ul class="menu-list">
-        {{#if canExpire}}
-          <li class="action">
+        {{#if this.auth.allowExpiration }}
+         <li class="action">
```

Comment:
I generally keep an indentation after an if statement.  Double check in the code and see if that's a consistent pattern, otherwise keep as is.